src/main/src/main/java/org/geoserver/catalog/DimensionInfo.java

```diff
@@ -122,4 +122,8 @@
      * #getAcceptableInterval()}.
      */
     public void setAcceptableInterval(String acceptableInterval);
+
+    public String getFixedValueRange();
```

Comment:
this get and the set method should be updated according to be getFixedValue and setFixedValue